### DISCUSSION TITLE
GitHub Student Developer Pack pending for 5+ days

### DISCUSSION THREAD

**Discussion by Programmer-0-0** [community/community#22117] [New to GitHub]
<p>Hi,<br>
I’ve submitted my request ( applied) for the GitHub student developer pack 6 days ago but my application or request is still pending.<br>
<img src="https://user-images.githubusercontent.com/67505416/181068920-fca9a0e7-f8e5-4107-a3f0-75a0c9b6a286.png" alt="Capture" data-base62-sha1="bvOFVb0hvdhcyTXZQDwYQKs4bST" width="372" height="246"><br>
I didn’t use a school issued email ( as my school doesn’t provide them) but I uploaded a valid document with my schools name and my name and also the issued date on it. Also, I’ve submitted the optional data for my schools location.<br>
Yet, it has been 6 days since I’ve submitted the form ( on June 27) and now its July 3. I know that the GitHub team must be busy and pressured due to COVID-19 but I researched on the pending time and found that the pending time ( for some ) can take up to 3 months and my school is to resume in September and I’m trying to learn how to code with python and C# as much as I could before that happens. I hope you consider looking at my request.</p>
<p>Thanks in advance !</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>Unfortunately if your application couldn’t be automatically approved using an accepted academic email address, you will have to wait while it is manually reviewed by one of our education specialists.</p>
<p>You can still get started learning though! Then you’ll be really ready when you’re approved. <a href="https://guides.github.com/activities/hello-world/">https://guides.github.com/activities/hello-world/</a></p>

**Comment by Programmer-0-0** [comment]
<p>Do you ( by any chance) have any idea of the estimate time that I have to wait until a specialist manually reviews my request.<br>
Thank you.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for your patience! Our education team will reach out once your request review has been completed; please keep an eye out for that contact.</p>

**Comment by Studentind** [comment]
GitHub is not a Beneficial for Students any more it's a Mental Torture to Student they F...ing Aproved application in purple 🟣 and after some hour it saying application expired please re submit 🤬
Now a days applications are not Aproved in Green they say Pending 🤬🤬🤬
